config/notifiers.go

```diff
@@ -120,12 +120,16 @@ type EmailConfig struct {
 	NotifierConfig `yaml:",inline"`
 
 	// Email address to notify.
-	To         string            `yaml:"to"`
-	From       string            `yaml:"from"`
-	Smarthost  string            `yaml:"smarthost,omitempty"`
-	Headers    map[string]string `yaml:"headers"`
-	HTML       string            `yaml:"html"`
-	RequireTLS bool              `yaml:"require_tls"`
+	To           string            `yaml:"to"`
+	From         string            `yaml:"from"`
+	Smarthost    string            `yaml:"smarthost,omitempty"`
+	AuthUsername string            `yaml:"auth_username"`
```

Comment:
These all have the same prefix. Would it be worthwhile to move them into their own YAML block?